include/evmc/loader.h

```diff
@@ -21,7 +21,10 @@ extern "C" {
 /** The function pointer type for EVMC create functions. */
 typedef struct evmc_vm* (*evmc_create_fn)(void);
 
-/** Error codes for the EVMC loader. */
+/// Error codes for the EVMC loader.
```

Comment:
`///` for comments are not used anywhere else in this file

Reply:
I noticed, but I wanted to use `//` everywhere since we enabled C99.